tmt/steps/provision/__init__.py

```diff
@@ -106,6 +112,7 @@ def go(self):
                     plugin.go()
                     if isinstance(plugin, ProvisionPlugin):
                         plugin.guest().details()
+                    self.info('')
```

Comment:
Shall we enable the extra space only when multiple guests are provisioned? It makes the single-guest output a bit untidy.